src/components/petType.jsx

```diff
@@ -13,6 +11,28 @@ const PetType = ({
   PetTypes,
   shouldShow,
 }) => {
+  const handleChange = (changeEvent) => {
+    const target = changeEvent.nativeEvent.target;
+    const index = target.selectedIndex;
+    const selectedText = target[index].text;
+    const { name } = changeEvent.target;
+    const localProps = rest.formik;
```

Comment:
not blocking i'm guessing this isn't net new, but we should look for a better way to do this. If we add another field then we would have to add another line to clear the field. This should be a bit more robust. I'm thinking maybe formik has some way to clear form values.

Reply:
Yes , this is not new , just making it work for select component.

Reply:
resolving, when you run into something like this and don't have an answer, please reach out for help.